queue/abstract.lua

```diff
@@ -393,45 +388,46 @@ local function make_self(driver, space, tube_name, tube_type, tube_id, opts)
     return self
 end
 
+--- Release all session tasks.
+local function release_session_tasks(session_uuid)
+    local taken_tasks = box.space._queue_taken_2.index.uuid:select(session_uuid)
```

Comment:
Here you use key wrapped in parenthesis but below you use delete with braces. I think it should be unified.

Reply:
Updated.